### PR TITLE
Scripts for Windows host and update for new website

### PR DESCRIPTION
**Pull request by simonjayhawkins**
parking this here for now.

### CODE REVIEW

**Comment by simonjayhawkins** [comment]
@jorisvandenbossche currently doing a dummy run of release. was trying to use a docker volume so no need to clone on a windows machine. it's getting messy so going back to use the local cloned volume for easier access from the host. maybe a bit longer before I tag.

(i'm running low of data allowance till 22/6, previous release took 20GB out of 50GB/month within a couple of days, but fingers crossed this won't be an issue. just a heads up in case it does.)

**Comment by simonjayhawkins** [comment]
I think I will persevere with the independent volume after all to ensure no issues with line endings

**Comment by jorisvandenbossche** [comment]
Let us now if it works out with your laptop/data. Otherwise I can also give it a try tomorrow or the day after.

File: Makefile [hunk 1]
```diff
@@ -135,9 +135,11 @@ wheels:
 
 
 download-wheels:
-	./scripts/download_wheels.py $(PANDAS_VERSION)
+	python ./scripts/download_wheels.py $(PANDAS_VERSION)
```
Comment (simonjayhawkins) [inline Makefile:138]: this was to run from git bash. change will be reverted

File: Dockerfile [hunk 2]
```diff
@@ -6,5 +6,3 @@ RUN apt-get update && \
 RUN conda update conda -y \
  && conda install -y conda-build conda-verify gcc_linux-64 gxx_linux-64 \
  && conda clean --all
- 
-COPY pandas /pandas
```
Comment (simonjayhawkins) [inline Dockerfile:10]: I think not needed

File: Makefile [hunk 3]
```diff
@@ -135,9 +135,11 @@ wheels:
...
 	# TODO: Fetch from https://www.lfd.uci.edu/~gohlke/pythonlibs/
 
 
 upload-pypi:
 	twine upload pandas/dist/pandas-$(PANDAS_VERSION)*.{whl,tar.gz} --skip-existing
+
+# twine upload pandas/dist/pandas-1.0.4*.{whl,tar.gz} --skip-existing
```
Comment (simonjayhawkins) [inline Makefile:145]: the regex pattern didn't work (can't remember whether was using git bash or Anaconda prompt. Now that using Docker can revert.

File: Makefile [hunk 4]
```diff
@@ -15,14 +15,12 @@ SHELL := /bin/bash
 
 init-repos:
 	git clone https://github.com/pandas-dev/pandas                   && git -C pandas           remote rename origin upstream && git -C pandas 		     remote add origin https://github.com/$(GH_USERNAME)/pandas
-	git clone https://github.com/pandas-dev/pandas-website           && git -C pandas-website   remote rename origin upstream && git -C pandas-website   remote add origin https://github.com/$(GH_USERNAME)/pandas-website
 	git clone https://github.com/conda-forge/pandas-feedstock        && git -C pandas-feedstock remote rename origin upstream && git -C pandas-feedstock remote add origin https://github.com/$(GH_USERNAME)/pandas-feedstock
```
Comment (simonjayhawkins) [inline Makefile:18]: didn't use this. I assume not now needed.

**Comment by TomAugspurger** [comment]
Thanks Simon, will give this a go for the RC.

**Pull request merged**